data/mods/DinoMod/monsters/zed-dinosaur_CBM.json

```diff
@@ -73,8 +73,8 @@
     "name": { "str": "Spinosaurus zombie bio-operator" },
     "copy-from": "mon_zpinosaurus",
     "diff": 5,
-    "proportional": { "hp": 2, "speed": 1.2 },
-    "relative": { "morale": 100, "melee_skill": 10, "melee_dice": 9, "dodge": 1, "armor_bash": 8, "armor_cut": 8, "armor_bullet": 8 },
+    "proportional": { "hp": 1.2, "speed": 1.15 },
+    "relative": { "melee_skill": 2, "melee_dice": 1, "dodge": 2, "armor_bash": 3, "armor_cut": 3, "armor_bullet": 3 },
     "flags": [
       "NO_BREATHE",
```

Comment:
Add PUSH_MON?

Reply:
Interesting, why?

Reply:
The non-zombie form has it, for one thing. Admittedly, one argument against it is that they've got a ranged attack - provided they can use it with other zombies in the way. (Otherwise, other zombies act as shields.)

Reply:
I'd like to hold off for now but it's an interesting idea to think about